src/storage/mvcc/txn.rs

```diff
@@ -201,7 +201,13 @@ impl<'a> MvccTxn<'a> {
                         Err(Error::Committed { commit_ts: ts })
                     }
                     // Rollbacked by concurrent transaction.
-                    None => Ok(()),
+                    None => {
```

Comment:
`get_txn_commit_ts` need return more information to indicate the txn is rollbacked or not.